### PR TITLE
Docker/Deployment

### PR DESCRIPTION
**Pull request by mtkp**
Add docker and deployment stuff.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,17 +21,36 @@ $ pip freeze > requirements.txt    # ensure dependencies are up to date
 $ deactivate                       # un-set the virtual environment
 ```
 
-## run the server
+### run the development server
 
 ```sh
-# assumes python packages installed
 $ ./run-server.sh             # starts the server listening on port 5000
```
Comment (jbeorse) [inline README.md]: Should be changed to $ ./run-server.bash

File: README.md [hunk 2]
```diff
@@ -21,17 +21,36 @@ $ pip freeze > requirements.txt    # ensure dependencies are up to date
...
-$ ./run-server.sh --gunicorn  # starts the server using gunicorn (single worker, port 5000)
 ```
 
-## run tests
+### run tests
 
 To run the server unit tests:
 ```sh
 $ ./run-tests.sh
```
Comment (jbeorse) [inline README.md]: Should be .bash, as above

File: README.md [hunk 3]
```diff
@@ -21,17 +21,36 @@ $ pip freeze > requirements.txt    # ensure dependencies are up to date
...
+
+1. Install elastic beanstalk CLI, `eb`, with pip: http://docs.aws.amazon.com/elasticbeanstalk/latest/dg/eb-cli3-getting-set-up.html#eb_cli3-install-with-pip
+2. Ensure you have IAM role keys in `~/.aws/credentials` (with elastic beanstalk permissions).
+3. Run `eb init` and pull down `lemongrab` from `us-west-2 : US West (Oregon)`.
+
+### deploying
+
+```sh
+# deploy the latest commit on the master branch
+$ ./deploy.sh
```
Comment (jbeorse) [inline README.md]: Should be .bash, as above

**Comment by jbeorse** [comment]
Looks good to me. I see you lost the game of "who can tolerate this awful yellow the longest" :-p

**Comment by mtkp** [comment]
I should never have played. It was truly unacceptable.

**Pull request merged**